edx_rest_framework_extensions/middleware.py

```diff
@@ -155,10 +155,12 @@ def _set_request_auth_type_guess_attribute(self, request):
                 auth_type = token_parts[0].lower()  # 'jwt' or 'bearer' (for example)
             else:
                 auth_type = 'other-token-type'
-        elif USE_JWT_COOKIE_HEADER in request.META and jwt_cookie_name() in request.COOKIES:
```

Comment:
**Note to Reviewer:** Since the consolidated jwt cookie with `jwt_cookie_name()` wouldn't exist if `USE_JWT_COOKIE_HEADER` didn't exist, it seems like this case was never needed.